plugin.py

```diff
@@ -93,6 +107,49 @@ def install_or_update(cls) -> None:
             shutil.rmtree(cls.basedir(), ignore_errors=True)
             raise
 
+    def on_pre_server_command(self, command: Mapping[str, Any], done_callback: Callable[[], None]) -> bool:
+
+        cargo_commands = []
+        for c in command["arguments"]:
+            if c["kind"] == "cargo":
+                cargo_commands.append(c)
+
+        if len(cargo_commands) == 0:
+            return False
+
+        window = sublime.active_window()
+        if window is None:
+            return False
+        view = window.active_view()
+        if not view:
+            return False
+        if not Terminus:
+            sublime.error_message(
+                'Cannot run executable "{}": You need to install the "Terminus" package and then restart Sublime Text'.format(output["kind"]))
+            return False
```

Comment:
You still want to return `True` in this situation, I believe, since if the server gets to handle that command, it will most likely throw an error. Also in the cases above.

Also, you have to call the `done_callback` when returning `True`.

Reply:
Ohhhh